deepwell/src/services/alias/service.rs

```diff
@@ -112,6 +115,11 @@ impl AliasService {
 
                     return Err(Error::Conflict);
                 }
+
+                if slug.bytes().len() < ctx.config().minimum_name_bytes {
+                    tide::log::error!("User's name does not contain enough bytes.");
```

Comment:
This could be more helpful. At least I think "not long enough" is a better error message. (This would need to be copied to the other locations as well.)
```suggestion
                    tide::log::error!(
                        "User's name is not long enough ({} < {})",
                        slug.bytes().len(),
                        ctx.config().minimum_name_bytes,
                    );
```